### PR TITLE
Backend cleaning and code documentation

### PR DESCRIPTION
**Pull request by Trivinyx**
This "Features" goal is to update the in code:
- Documentation of the backend 
- Remove unused imports where possible
- Delete unused functions, and functions without any current value.
- Remove in dev code comments etc where possible.

### CODE REVIEW

File: backend/img2mapAPI/routers/georefProject.py [hunk 1]
```diff
@@ -1,42 +1,62 @@
...
+    - Add a point to a project
+    - Update a point in a project
+    - Delete all points from a project
+    - Delete a point from a project
+    - Get all points of a project
+    - Get a point in a project by id
+    - Upload an image to a project
+    - Get the image of a project by id
+    - Georeference the image of a project by id
+    - Re-georeference the image of a project by id
```
Comment (ostepizza) [inline backend/img2mapAPI/routers/georefProject.py]: "    - Re-georeference the image of a project by id"
You remove this further down

File: backend/img2mapAPI/routers/georefProject.py [hunk 2]
```diff
@@ -1,42 +1,62 @@
...
+    - Get the georeferenced image of a project by id
+    - Get the corner coordinates of the image of a project by id
```
Comment (ostepizza) [inline backend/img2mapAPI/routers/georefProject.py]: "    - Get the corner coordinates of the image of a project by id"
Suggest to change this due to Sebs approved PR where corners are no longer used

**Review by sebmid — APPROVED** [review]
Lots of words lots of comments, very readable though, big imporvement
hum32lholr341

File: backend/main.py [hunk 3]
```diff
@@ -1,13 +1,26 @@
+""" Entry point for the backend server
+
+    Run the server with the following command:
+        python main.py
+    The scrip will load the environment variables from the .env file in the root directory
```
Comment (ostepizza) [inline backend/main.py]: scrip

**Review by sebmid — APPROVED** [review]
pogu

**Pull request merged**